src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/BlobManager.java

```diff
@@ -322,7 +328,7 @@ private void newMoveFileToRejectedContainer(
 
         try {
             inputBlob.deleteWithResponse(
-                DeleteSnapshotsOptionType.ONLY,
+                DeleteSnapshotsOptionType.INCLUDE,
```

Comment:
previous version we were using DeleteSnapshotsOption.NONE which `Specifies deleting the blob only. If the blob has snapshots, this option will result in an error from the service`
I think In new version it does not result in error if there is snapshots.  We can pass 'null'  it will delete just the blob. INCLUDE deletes all,  ONLY deletes only snapshots.